src/sagemaker_xgboost_container/algorithm_mode/serve.py

```diff
@@ -200,29 +207,31 @@ def _get_sparse_matrix_from_libsvm(payload):
 
 def _parse_content_data(request):
     dtest = None
-    content_type = serve_utils.get_content_type(request)
+    content_type = data_utils.get_content_type(request.content_type)
```

Comment:
For consistency with the code above, you should just call `get_content_type` since you've imported the function directly.